mobile/src/reducers/people/people.reducer.ts

```diff
@@ -0,0 +1,48 @@
+import { Reducer } from 'redux';
+import { Map, Set } from 'immutable';
+import { combineEpics } from 'redux-observable';
+import { NavigationAction } from 'react-navigation';
+
+import { User } from '../user/user.model';
+import { Employee } from '../organization/employee.model';
+import { OrganizationActions, loadEmployeesForDepartment } from '../organization/organization.action';
+import { UserActions } from '../user/user.action';
+import { PeopleActions } from './people.action';
+import { EmployeeMap } from '../organization/employees.reducer';
+
+export interface PeopleState {
+    employees: EmployeeMap;
+    employeesDepartmentSubsetFilterCallback: any;
```

Comment:
hold on. Why there are three different functions? There should be only one and you switch between them. Then you can reuse the same Component for Room and Department screens, the only thing that will change is a function returned.

Reply:
If I use one universal function - it's working in such way: once I open Room tab the content at Department tab became same - so new predicate (room specific) is applied in both cases.

Reply:
what